mypy/checker.py

```diff
@@ -2630,7 +2639,8 @@ def check_member_assignment(self, instance_type: Type, attribute_type: Type,
             # the return type of __get__. This doesn't match the python semantics,
             # (which allow you to override the descriptor with any value), but preserves
             # the type of accessing the attribute (even after the override).
-            rvalue_type = self.check_simple_assignment(get_type, rvalue, context)
+            rvalue_type = self.check_parent_member_assignment(instance_type, get_type,
+                                                              attribute_type, rvalue, context)
```

Comment:
Why do you need to pass _both_ `attribute_type` and `get_type` here? This looks wrong, I think you should only pass `get_type`.

Reply:
On a second thought, it seems to me, we may not show this additional info if descriptor protocol is involved, it may only add confusion, because the definition a user will see in the base class, is the descriptor type. But this is not a strong opinion.